config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

```diff
@@ -233,6 +233,45 @@ presubmits:
 
     annotations:
       testgrid-create-test-group: 'true'
+  - name: pull-kubernetes-e2e-gce-serial
+    always_run: false
+    optional: true
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
+    spec:
+      containers:
+      - args:
+        - --root=/go/src
+        - --repo=k8s.io/kubernetes=$(PULL_REFS)
+        - --repo=k8s.io/release
+        - --upload=gs://kubernetes-jenkins/pr-logs
+        - --timeout=150
+        - --scenario=kubernetes_e2e
+        - --
+        - --build=bazel
+        - --cluster=
+        - --extract=local
+        - --gcp-node-image=ubuntu
+        - --image-family=ubuntu-gke-1804-lts-1
+        - --image-project=ubuntu-os-gke-cloud
+        - --gcp-zone=us-west1-b
+        - --provider=gce
+        - --runtime-config=batch/v2alpha1=true
+        - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-serial
+        - --test_args=--ginkgo.focus=\[Driver:.*\].*(\[Serial\]|\[Disruptive\]) --ginkgo.skip=\[Flaky\] --minStartupPods=8
```

Comment:
This is going to run almost all the tests here which will probably take 4 hours and hit the timeout. https://k8s-testgrid.appspot.com/sig-storage-kubernetes#gce-serial

One reason is because all pd CSI driver tests have to run as serial. Making the focus just "Disruptive" may help reduce those the tests. Plus the CSI driver serial tests are already run under the CSI serial job above

Reply:
done, `--ginkgo.focus=\[sig-storage\].*\[Disruptive\] --ginkgo.skip=\[Flaky\]`

It will run 399 of 4980 Specs.